packages/browser/package.json

```diff
@@ -42,7 +42,7 @@
     "vitest": ">=0.29.4"
   },
   "dependencies": {
-    "modern-node-polyfills": "0.1.0",
+    "modern-node-polyfills": "^0.1.1",
```

Comment:
@Aslemammad Did you remove `^` on purpose? Is it fine to add `^`?

Reply:
Actually, I do not remember, but I feel it is ok and no issue with it.